packages/odyssey-react-mui/src/ui-shell/renderUiShell.tsx

```diff
@@ -73,8 +73,11 @@ export const renderUiShell = ({
   | "sideNavBackgroundColor"
   | "topNavBackgroundColor"
 >) => {
-  const appRootElement =
-    explicitAppRootElement || document.createElement("div");
+  let appRootElement = explicitAppRootElement;
+  if (!appRootElement) {
+    appRootElement = document.createElement("div");
+    appRootElement.style.height = "inherit";
+  }
```

Comment:
I don't like mutations. For CSS, I'd rather use `setProperty` which also avoids an explicit mutation even though it still modifies the CSS:

```suggestion
  const appRootElement =
    explicitAppRootElement || document.createElement("div");

  if (!explicitAppRootElement) {
    appRootElement.style.setProperty("height", String(height) || "inherit");
  }
```

The `height` variable is something I added to my branch.

We might want to use `"100%"` as is the case for Enduser in my testing. I'll be doing more testing on my end to see if this works without the need for `"100%"`, but if I modified `OdysseyProvider` and `OdysseyThemeProvider` to have `"100%"`, it fixed the height issues in Enduser.

Reply:
This is my PR:
https://github.com/okta/odyssey/pull/2508

I'm adding your change.

Reply:
Actually, I don't think we can set this. If we use `100%` here, it's going to get overridden anyway. We're setting this to a fixed height in UI Shell.